prow/cmd/deck/main.go

```diff
@@ -585,55 +595,77 @@ func renderSpyglass(sg *spyglass.Spyglass, ca *config.Agent, src string, o optio
 // Query params:
 // - name: required, specifies the name of the viewer to load
 // - src: required, specifies the job source from which to fetch artifacts
-func handleArtifactView(sg *spyglass.Spyglass, ca *config.Agent) http.HandlerFunc {
+func handleArtifactView(o options, sg *spyglass.Spyglass, ca *config.Agent) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
-		start := time.Now()
+		//start := time.Now()
```

Comment:
hmm why is this still here, but commented out?

Reply:
Commented out a bunch of code for refactoring, had to comment this out to avoid the unused variable error, deleted all the other code when I was done, forgot to also delete this. Will remove.